### PR TITLE
Create dev build for generic helm

### PR DESCRIPTION
**Pull request by hitesh-wani-px**
**What this PR does / why we need it**:
This will add dev build of generic helm in feature branch PWX-29986
**Which issue(s) this PR fixes** (optional)
Closes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by hitesh-wani-px** [comment]
**Steps for installing PX with generic helm dev version**
1. Add helm repo
```
helm repo add portworx https://raw.githubusercontent.com/portworx/helm/PWX-29986/stable/
```
2. Install PX with dev version
```
helm install my-release --set internalKVDB=true ./charts/portworx/ -n portworx --version 3.1.1-PWX-29986-dev
```

File: charts/portworx/Chart.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 name: portworx
-version: 3.1.0
+version: 3.1.1-PWX-29986-dev
```
Comment (diptiranjanpx) [inline charts/portworx/Chart.yaml:2]: you have to stick to 3.x.x as the version and I think we decided it to be `3.1.2`

Comment (hitesh-wani-px) [reply charts/portworx/Chart.yaml]: changed it 3.1.2

File: stable/index.yaml [hunk 2]
```diff
@@ -1,9 +1,34 @@
...
+    - persistent storage
+    - portworx
+    - amd64
+    - Commercial
+    kubeVersion: '>=1.21.0-0'
+    name: portworx
+    sources:
+    - https://github.com/portworx/helm
+    urls:
+    - https://raw.githubusercontent.com/portworx/helm/dev-build/stable/portworx-3.1.1.tgz
```
[the remark this answers is not in the document]
Comment (hitesh-wani-px) [reply stable/index.yaml]: Yes I manually edited it.
Now it is auto generated with helm cmd. I pointed it to `PWX-29986` as we are merging this `PWX-29986`  feature branch.

**Review by diptiranjanpx — APPROVED** [review]

**Pull request merged**